Marlin/Configuration.h

```diff
@@ -3066,6 +3066,7 @@
     #define NEOPIXEL2_PIXELS           15 // Number of LEDs in the second strip
     #define NEOPIXEL2_BRIGHTNESS      127 // Initial brightness (0-255)
     #define NEOPIXEL2_STARTUP_TEST        // Cycle through colors at startup
+    //#define NEOPIXEL_M150_DEFAULT_S0    // By default M150 without 'S' applies to all strips. Enable to set the first strip only.
```

Comment:
Maybe an option to affect a particular index by default would be better, instead of requiring the default one to be the first index. That will make it a little more flexible.